filter-processor/README.adoc

```diff
@@ -0,0 +1,31 @@
+= Spring Cloud Stream Filter Processor
+
+A *Spring Cloud Stream* module that allows one to discard or retain messages according to a predicate.
+== Requirements
+
+To run this module, you will need to have installed:
```

Comment:
I really think we can leave this line out. I tried to establish the simplest format for the READMEs in time-source and log-sink, e.g.:  https://github.com/spring-cloud/spring-cloud-stream-modules/tree/master/log-sink